common/font.c

```diff
@@ -376,3 +376,49 @@ void fontExit()
     if (s_font_libret==0) FT_Done_FreeType(s_font_library);
 }
 
+/*Automatically gives you the desired x-coordinate 
+ *based on the string length and desired alignmenr
+ *rY=reference point... where to align around
+ *align='t','b','c' translates too (top,bottom,center)
+ *'t' aligned, top of text aligns with rY, 
+ *you get the rest....
+ */
+uint32_t getYCoordinate(u32 font,uint32_t rY,const char* text, const char align){
+    uint32_t height_o,width;
+    GetTextDimensions(font,text,&width,&height_o);
+    uint32_t height = (uint32_t)height_o;
+    uint32_t fC = (rY-height);
+    switch(align){
```

Comment:
Insert newline + space.

Reply:
insert a newline right after the switch statement? or before?

Reply:
done